libbeat/dashboards/dashboards.go

```diff
@@ -59,6 +62,29 @@ func ImportDashboards(
 
 	return setupAndImportDashboardsViaKibana(ctx, hostname, kibanaConfig, &dashConfig, msgOutputter)
 
+	// Generate index pattern
+	version, _ := common.NewVersion("7.0.0") // TODO: dynamic version
+	//beatVersion, _ := common.NewVersion("7.0.0") // TODO: dynamic version
+	//indexP, _ := esConfig.String("index", 0)
```

Comment:
@urso I think this also ties into your current work around indexing changes. Where should the config which index pattern should be used go for Kibana setup part?